packages/manager/apps/web/client/app/hosting/cdn/shared/add-edit-cache-rule/hosting-shared-add-edit-cache-rule.routing.js

```diff
@@ -21,7 +21,7 @@ export default /* @ngInject */ ($stateProvider) => {
     rule: null,
   };
 
-  $stateProvider.state('app.hosting.cdn.shared.addCacheRule', {
+  $stateProvider.state('app.hosting.dashboard.cdn.shared.addCacheRule', {
```

Comment:
SHould be fixed on `master` maybe ?

Reply:
Currently the add/edit cache rule action is opened by an `$uibModal`, so it is working.
It is a fix as I have make some changes to state declaration, and remove `$uibModal`.
So it can wait a release